content/en/blog/_posts/2024-03-08-mid-cycle-1.30.md

```diff
@@ -0,0 +1,72 @@
+
+### Structured Authorization Configuration ([KEP-3221](https://kep.k8s.io/3221))
+
+The Kubernetes API server's authorizer chain can now be configured using a configuration file. This [feature](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/3221-structured-authorization-configuration) enables the creation of authorization chains with multiple webhooks with well-defined parameters that validate requests in a particular order and allows fine-grained control – such as explicit Deny on failures. The configuration file allows one to specify CEL rules to pre-filter requests before they are dispatched to webhooks to prevent unnecessary invocations. API Server will also automatically reload the authorizer chain when the configuration file is modified.
+
+An example configuration with all possible values is provided in the [Authorization docs](/docs/reference/access-authn-authz/authorization/#configuring-the-api-server-using-an-authorization-config-file). You must specify the path to that authorization configuration using the `--authorization-config` command line argument. While the feature is in the Alpha or Beta stage, there is no change if you want to keep on using command line flags instead of using a configuration file. From Kubernetes 1.30, the feature would be turned on by default. For more details, read the [Authorization docs](/docs/reference/access-authn-authz/authorization/#configuring-the-api-server-using-an-authorization-config-file).
+
+### CEL for Admission Control ([KEP-3488](https://kep.k8s.io/3488))
+
+Integrating Common Expression Language (CEL) for admission control in Kubernetes introduces a more dynamic and expressive way of evaluating admission requests. This feature allows complex, fine-grained policies to be defined and enforced directly through the Kubernetes API, enhancing security and governance capabilities without compromising performance or flexibility.
+
+CEL's addition to Kubernetes admission control empowers cluster administrators to craft intricate rules that can evaluate the content of API requests against the desired state and policies of the cluster without resorting to Webhook-based access controllers. This level of control is crucial for maintaining the integrity, security, and efficiency of cluster operations, making Kubernetes environments more robust and adaptable to various use cases and requirements.
+
+### Container Resource based Pod Autoscaling ([KEP-1610](https://kep.k8s.io/1610))
+
+HPA ContainerResource type metric graduates to stable in v1.30. It allows us to configure Autoscaling based on resource usage of individual containers. See [our previous blog](https://kubernetes.io/blog/2023/05/02/hpa-container-resource-metric/) for further details.
```

Comment:
Consider using the site-relative link for the blog URL.

```suggestion
HPA ContainerResource type metric graduates to stable in v1.30. It allows us to configure Autoscaling based on resource usage of individual containers. 
See [our previous blog](/blog/2023/05/02/hpa-container-resource-metric/) for further details.
```

Reply:
Thank you! Are the site-relative links documented somewhere (we would like to add the usage guidelines for these to our handbook)?